php7.2-stretch/Dockerfile

```diff
@@ -0,0 +1,174 @@
+FROM php:7.2-fpm-stretch AS build
+
+ENV PATH=/root/.local/bin:$PATH \
+    PROCFILE_PATH=/app/Procfile \
+    LC_ALL=C.UTF-8 \
+    NVM_DIR=/opt/nvm \
+    NVM_VERSION=0.34.0
+
+USER root
+
+RUN echo "force-unsafe-io" > /etc/dpkg/dpkg.cfg.d/02apt-speedup
+RUN echo "Acquire::http {No-Cache=True;};" > /etc/apt/apt.conf.d/no-cache
+
+RUN DEBIAN_FRONTEND=noninteractive dpkg-reconfigure -f noninteractive tzdata
+RUN apt-get update && apt-get upgrade -y
+
+RUN DEBIAN_FRONTEND=noninteractive apt-get install -yqq \
```

Comment:
`DEBIAN_FRONTEND=noninteractive` can be added to the env vars. Is it still required though?

Reply:
There are a few php packages that interactively ask for additional information if this is not set